frontend/src/App.tsx

```diff
@@ -940,7 +940,7 @@ export class App extends PureComponent<Props, State> {
           attach={window}
           focused={true}
         >
-          <div className={outerDivClass}>
+          <StyledApp className={outerDivClass}>
```

Comment:
the className can be killed in conjunction with cleanup that can be done for the header PR